src/views/esp/view_esp.cpp

```diff
@@ -69,6 +69,85 @@ namespace big
 			if (distance < g.esp.box_render_distance[1] && distance > g.esp.box_render_distance[0] && g.esp.box)
 				draw_list->AddRect({esp_x - (62.5f * multplr), esp_y - (175.f * multplr)}, {esp_x - (62.5f * multplr) + (125.f * multplr), esp_y - (175.f * multplr) + (350.f * multplr)}, esp_color);
 
+			if (distance < g.esp.skeleton_render_distance[1] && distance > g.esp.skeleton_render_distance[0] && g.esp.skeleton)
+			{
+				const std::vector<PedBones> bone_ids = // size: 18
+				{
+					PedBones::SKEL_Head, 
+					PedBones::SKEL_Neck_1,
+					PedBones::SKEL_Spine_Root, 
+					PedBones::SKEL_Pelvis, 
+					PedBones::SKEL_L_Thigh, 
+					PedBones::SKEL_R_Thigh, 
+					PedBones::SKEL_L_Calf, 
+					PedBones::SKEL_R_Calf, 
+					PedBones::SKEL_L_Foot, 
+					PedBones::SKEL_R_Foot, 
+					PedBones::SKEL_L_Clavicle, 
+					PedBones::SKEL_R_Clavicle, 
+					PedBones::SKEL_L_UpperArm, 
+					PedBones::SKEL_R_UpperArm, 
+					PedBones::SKEL_L_Forearm, 
+					PedBones::SKEL_R_Forearm, 
+					PedBones::SKEL_L_Hand, 
+					PedBones::SKEL_R_Hand
+				};
+
+				std::vector<rage::fvector4> bones = {};
+				const size_t skel_size            = 18;
+
+				for (int i = 0; i < skel_size; i++)
```

Comment:
Maybe if you fucking wrote it in English I could fucking understand it.

Reply:
Calm, there's no need for aggression mate. My english is fine. 

Ask "wdym" next time and I'll try to give you a more indepth explanation. Let's try to keep a friendly tone between one another. 👍